src/main/target/RUSH_BLADE_F7/target.h

```diff
@@ -15,40 +15,67 @@
  * along with INAV.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-
 #pragma once
 
+#ifdef RUSH_BLADE_F7
 #define TARGET_BOARD_IDENTIFIER "RBF7"
 #define USBD_PRODUCT_STRING  "RUSH_BLADE_F7"
+#endif
 
-#define LED0                    PB10  //Blue   SWCLK
-// #define LED1                    PA13  //Green  SWDIO
+#ifdef RUSH_BLADE_F7_HD
+#define TARGET_BOARD_IDENTIFIER "RBF7HD"
+#define USBD_PRODUCT_STRING     "RUSH_BLADE_F7_HD"
+#endif
 
+#define LED0                    PB10
 #define BEEPER                  PB2
 #define BEEPER_INVERTED
 
-// *************** SPI1 Gyro & ACC *******************
+// *************** SPI **********************
 #define USE_SPI
+
 #define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PC4
 #define SPI1_SCK_PIN            PA5
 #define SPI1_MISO_PIN           PA6
 #define SPI1_MOSI_PIN           PA7
 
+#define USE_SPI_DEVICE_2
+#define SPI2_NSS_FLASH_PIN      PB12
+#define SPI2_NSS_OSD_PIN        PB11
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+
+// *************** Gyro & ACC **********************
+#define USE_EXTI
```

Comment:
We don't use EXTI anymore

Reply:
Missed that memo! Cleaned up basing on 60b810c.